jdk/test/sun/security/pkcs11/fips/NssdbPin.java

```diff
@@ -0,0 +1,350 @@
+            throw new Exception("Unsupported property type.");
+        }
+        if (DEBUG) {
+            p.inheritIO();
+            p.prop("java.security.debug", "sunpkcs11");
+            p.debug(NssdbPin.class.getName());
+
+            // Need the launched process to connect to a debugger?
+            //System.setProperty("test.vm.opts", "-Xdebug -Xrunjdwp:" +
+            //         "transport=dt_socket,address=localhost:8000,suspend=y");
+        } else {
+            p.nodump();
+        }
+        procCb.accept(p);
+        p.start().waitFor(0);
+    }
+
+    private static String escapeForPropsFile(String str) throws Throwable {
+        StringBuffer sb = new StringBuffer();
+        for (int i = 0; i < str.length(); i++) {
+            int cp = str.codePointAt(i);
+            if (Character.UnicodeBlock.of(cp)
+                    == Character.UnicodeBlock.BASIC_LATIN) {
+                sb.append(Character.toChars(cp));
+            } else {
+                sb.append("\\u").append(String.format("%04X", cp));
+            }
+        }
+        return sb.toString();
+    }
+
+    private static void createNSSDB(TestContext ctx) throws Throwable {
+        ProcessBuilder pb = getModutilPB(ctx, "-create");
+        if (DEBUG) {
+            System.out.println("Creating an NSS DB in " + ctx.workspace +
+                    "...");
+            System.out.println("cmd: " + String.join(" ", pb.command()));
+        }
+        if (pb.start().waitFor() != 0) {
+            throw new Exception("NSS DB creation failed.");
+        }
+        generatePinFile(ctx);
+        pb = getModutilPB(ctx, "-changepw", NSSDB_TOKEN_NAME,
+                "-newpwfile", ctx.nssdbPinFile.toString());
+        if (DEBUG) {
+            System.out.println("NSS DB created.");
+            System.out.println("Changing NSS DB PIN...");
+            System.out.println("cmd: " + String.join(" ", pb.command()));
+        }
+        if (pb.start().waitFor() != 0) {
+            throw new Exception("NSS DB PIN change failed.");
+        }
+        if (DEBUG) {
+            System.out.println("NSS DB PIN changed.");
+        }
+    }
+
+    private static ProcessBuilder getModutilPB(TestContext ctx, String... args)
+            throws Throwable {
+        ProcessBuilder pb = new ProcessBuilder("modutil", "-force");
+        List<String> pbCommand = pb.command();
+        if (args != null) {
+            pbCommand.addAll(Arrays.asList(args));
+        }
+        pbCommand.add("-dbdir");
+        pbCommand.add(ctx.nssdbPath);
+        if (DEBUG) {
+            pb.inheritIO();
+        } else {
+            pb.redirectError(ProcessBuilder.Redirect.INHERIT);
+        }
+        return pb;
+    }
+
+    private static void generatePinFile(TestContext ctx) throws Throwable {
+        ctx.nssdbPinFile = Files.createTempFile(ctx.workspace, null, null);
+        try (BufferedWriter bw = Files.newBufferedWriter(ctx.nssdbPinFile,
+                new OpenOption[] {})) {
```

Comment:
Since we aren't passing any special option, we don't need to pass an empty array when calling a varargs method, we can just use `Files.newBufferedWriter(ctx.nssdbPinFile)` and remove the `import java.nio.file.OpenOption` statement.

Reply:
Good catch!